consensus/state.go

```diff
@@ -729,7 +756,8 @@ func (cs *ConsensusState) EnterPrevoteWait(height int, round int) {
 
 	// After `timeoutPrevote0+timeoutPrevoteDelta*round`, EnterPrecommit()
 	go func() {
-		time.Sleep(timeoutPrevote0 + timeoutPrevote0*time.Duration(round))
+		time.Sleep(timeoutPrevote0 + timeoutPrevoteDelta*time.Duration(round))
+		cs.timeoutChan <- TimeoutEvent{RoundStepPrevote, height, round}
 		cs.EnterPrecommit(height, round)
```

Comment:
Ok question. Suppose we want to fire the timeout event from EnterPrevote. Isn't it possible that we timeout out of Propose, but by the time we finish the checks in EnterPrevote we have the complete proposal? Seems like might just be best to pass the timeout boolean after all